blueprint/src/content.tex

```diff
@@ -203,9 +264,36 @@ \section{MainTheorem}
 \begin{lemma}\label{char_fun_FinInter}
   \lean{char_fun_FinInter}
   \uses{FinInter₀,eq_FinInter₀,toInt,char_fun}
+  \leanok
   The characteristic function of the intersection of a family of finite sets indexed by \verb|β| is equal to the product of their characteristic functions.
 \end{lemma}
 
+\begin{proof}
+  \leanok
+  Let $\mathcal{F} = \{A_1, A_2, \ldots, A_n\}$ be a collection of sets and let $B = \bigcap_{A \in \mathcal{F}} A$ be the intersection of these sets. Consider the characteristic function $\chi_B(x)$ for any element $x$. We wish to establish the following equivalence:
+
+  \begin{equation*}
+  \chi_B(x) = \prod_{A \in \mathcal{F}} \chi_A(x)
+  \end{equation*}
+
+  We analyze two cases:
+
+  Case 1: $x$ belongs to the intersection $B$
+
+  \paragraph{Left Hand Side (LHS):} $\chi_B(x) = 1$. This condition is equivalent to $x$ belonging to every set $A \in \mathcal{F}$.
```

Comment:
Do not use paragraph either. This makes the html quite ugly.
![image](https://github.com/user-attachments/assets/073ad3a4-0d71-46a2-a1a6-4be4b5f388c3)